src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -623,6 +630,8 @@ namespace AppInstaller::CLI::Workflow
                 installContext.SetTerminationHR(Workflow::HandleException(installContext, std::current_exception()));
             }
 
+            RefreshPathVariableForCurrentProcess();

```

Comment:
This should only be called after each dependency package installation. InstallMultiplePackages will be used by upgrade -all, import etc which in this implementation, an unnecessary amount of path refresh will be performed.

Reply:
added a new parameter to InstallMultiplePackages that controls whether path is refreshed. This is only set to true for package dependencies and the default is set to false.